radio/src/translations/de.h.txt

```diff
@@ -921,22 +921,22 @@
 #define TR_MOVE_MODEL          "Verschiebe Modell"
 #define TR_BACKUP_MODEL        "Modell auf SD-Karte"  //9XR-Pro
 #define TR_DELETE_MODEL        "Modell löschen" // TODO merged into DELETEMODEL?
-#define TR_RESTORE_MODEL       "Modell Restore"
+#define TR_RESTORE_MODEL       "Modell wiederherstellen"
 #define TR_SDCARD_ERROR        "SD-Kartenfehler"
 #define TR_NO_SDCARD           "Keine SD-Karte"
 #define TR_SDCARD_FULL         "SD-Karte voll"
 #define TR_INCOMPATIBLE        "Nicht kompatibel"
 #define TR_WARNING             "WARNUNG"
 #define TR_EEPROMWARN          "EEPROM"
-#define TR_STORAGE_WARNING     "STORAGE"
-#define TR_EEPROM_CONVERTING   "EEPROM Converting"
+#define TR_STORAGE_WARNING     "SPEICHER"
+#define TR_EEPROM_CONVERTING   "EEPROM Konvertierung"
 #define TR_THROTTLEWARN        "GAS"
 #define TR_ALARMSWARN          "ALARM"
 #define TR_SWITCHWARN          "SCHALTER"
 #define TR_FAILSAFEWARN        "FAILSAFE"
-#define TR_WRONG_SDCARDVERSION "Expected version: "
-#define TR_WRONG_PCBREV        "Wrong PCB detected"
-#define TR_EMERGENCY_MODE      "EMERGENCY MODE"
+#define TR_WRONG_SDCARDVERSION "Erwartete Version: "
+#define TR_WRONG_PCBREV        "Falsches PCB detektiert"
+#define TR_EMERGENCY_MODE      "NOTFALL MODUS"
 #define TR_PCBREV_ERROR        "PCB error"

```

Comment:
I will use PCB Problem since that is the heading for "Falsches PCB erkannt" and I think "Problem" fits better here.